client/tests/integration/triggers/by_call_trigger.rs

```diff
@@ -121,10 +124,11 @@ fn trigger_failure_should_not_cancel_other_triggers_execution() -> Result<()> {
             bad_trigger_instructions,
             Repeats::Indefinitely,
             account_id.clone(),
-            TriggeringFilterBox::ExecuteTrigger(ExecuteTriggerEventFilter::new(
-                bad_trigger_id.clone(),
-                account_id.clone(),
-            )),
+            TriggeringEventFilterBox::ExecuteTrigger(
```

Comment:
```suggestion
            TriggeringEventFilterBox::from(
```

Reply:
I would rather address this in a better way by resolving #4259